doc/release/0.16.0-notes.rst

```diff
@@ -94,6 +119,8 @@ The Wishart distribution and its inverse have been added, as
 The Exponentially Modified Normal distribution has been
 added as `scipy.stats.exponnorm`.
 
```

Comment:
Maybe also mention `random_state` parameter for stats.distributions, both univariate and multivariate.